server/views/layout.njk

```diff
@@ -31,10 +31,12 @@
 {% block beforeContent %}
   {{ govukPhaseBanner({
     tag: {
-      text: "alpha"
+      text: "beta"
     },
     html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
   }) }}
+
+  {% include 'partials/formError.njk' %}
 {% endblock %}
```

Comment:
I wonder if this might fail accessibility testing being, like the phase banner, outside of HTML landmarks. I'll try it and pick this up in my PR.

Reply:
Wave has no issue with this, but thats not to say Axe wont :) 

![Screenshot 2021-09-03 at 11 05 35](https://user-images.githubusercontent.com/30229564/131988830-9f397ddd-510e-4d4e-a5c3-4a2b1b6dd28a.png)